packages/generate/README.md

```diff
@@ -35,7 +35,7 @@ $ npx @edgedb/generate <generator> [options]
 **Deno**
 
 ```bash
-$ deno run --allow-all --unstable https://deno.land/x/edgedb/generate.ts <generator> [options]
+$ deno run --allow-all --unstable npm:@edgedb/generate <generator> [options]
```

Comment:
Do we still need the `--unstable` flag? I guess `--allow-all` is needed for fs access.

Reply:
u are right, we don't need it, I was testing it in Deno example project without` --unstable` flag and it worked